docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/api/index.md

```diff
@@ -0,0 +1,144 @@
+
+## Getting started
+
+You can have a look at and interact with all available endpoints in the [API documentation](https://console.snowplowanalytics.com/api/msc/v1/docs).
+
+### Authorizing in the API documentation
+
+To be able to post sample requests in the documentation you need to click the `Authorize` button at the top of the document and authorize with your token. The value for the token field in each individual requests is overwritten by this authorization.
+
+The endpoints focus on the main operations in the workflow around:
+
+1. Retrieving existing data products and their event specifications, also known as tracking scenarios in the current v1 API version
+2. Creating or editing new or existing data products
+3. Seeing the data product history
+4. Managing user subscriptions
+
+Each request will need to include your company's `organizationID` which is a UUID that can be retrieved from the URL immediately following the .com when visiting console:
+
+![](images/orgID.png)
+
+```mdx-code-block
+import GetConsoleApiKey from "@site/docs/reusable/get-console-api-key/_index.md"
+
+<GetConsoleApiKey/>
+```
+
+## Retrieving Information about Data Products
+
+The following `GET` requests are designed to allow you to access information about data products. These data products could be stored in either your development environment registry or your production environment registry.
+
+### Retrieve a List of All Data Products
+
+To retrieve a comprehensive list of all data products in your organization, you can use the following GET request:
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1`
+
+Query parameter `organizationId` is required.
+
+### Retrieving Information about a Specific Data Product
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}`
+
+Query parameters `organizationId` and `dataProductId` are required.
+
+### Retrieve History Information for a Data Product
+
+If you wish to retrieve information about a specific data product, you can use the following GET request:
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}/history`
+
+You can pass several parameters to control the result of the response:
+
+- **before**: returns records equal or less than the timestamp in the ISO-8601 format
+- **limit**: limits the number of records
+- **offset**: skip the first N results
+- **order**: order of returned records, `asc` or `desc`. Defaults to `desc`.
+
+Query parameter `organizationId` is required.
+
+## Creating and updating Data Products
+
+### Creating a Data Product
+
+This `POST` request allows you to create a new data product within an organization.
+
+`**POST** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1`
+
+The request body is mandatory and should be in JSON format. Here’s an example of how the request body might look:
```

Comment:
The minimum payload would be a json with only the `name` of the data product. There rest of the fields are optional and not required on creation.

Reply:
done - thanks